src/main/java/org/openhab/tools/analysis/checkstyle/api/AbstractStaticCheck.java

```diff
@@ -227,4 +220,21 @@ protected void logMessage(String filePath, int line, String fileName, String mes
         fireErrors(filePath);
         dispatcher.fireFileFinished(filePath);
     }
+
+    /**
+     * Parsed the content of a markdown file.
+     *
+     * @param fileText - Represents the text contents of a file
+     * @param blockTypes - the enabled block types
+     * @return The markdown node
+     */
+    protected Node parseMarkdown(FileText fileText, Set<Class<? extends Block>> blockTypes) {
```

Comment:
Where does this method come from?

Reply:
It comes from the MarkdownCheck. I extracted it here in case someone needs to parse a markdown file in the future.